health/health.d/megacli.conf

```diff
@@ -67,5 +76,8 @@ component: RAID
     every: 10s
      warn: $this >= 100
      crit: $this >= 500
-     info: average battery backup unit (BBU) charge cycles count over the last 10 seconds
+     info: Average battery backup unit (BBU) charge cycles count over the last 10 seconds. \
```

Comment:
If you accept to change the three initial letter for uppercase, please, change here too.